ledger-core/v2/virtual/handlers/handle_meta.go

```diff
@@ -103,6 +103,16 @@ func FactoryMeta(message *statemachine.DispatcherMessage) (pulse.Number, smachin
 			ctx.SetTracerID(traceID)
 			return &SMVDelegatedCallRequest{Meta: payloadMeta, Payload: obj}
 		}
+	case *payload.VDelegatedCallResponse:
+		pl := payload.VDelegatedCallResponse{}
```

Comment:
it is already unmarshalled and lives in obj variable

Reply:
Fix it